File: 3379-score-of-a-string/score-of-a-string.c
int scoreOfString(char* s) {
    int i,sum=0,a1,a2;
    for(i=0;s[i]!='\0' && s[i+1]!='\0';i++)
    {
        a1=(int)s[i];
        a2=(int)s[i+1];
        sum=sum+abs(a1-a2);
    }
    return sum;
}

File: 3379-score-of-a-string/score-of-a-string.java
class Solution {
    public int scoreOfString(String s) {
        int sum=0;
        for(int i=0;i<s.length()-1;i++)
        {
            int ascii1 = (int)s.charAt(i);      
            int ascii2 = (int)s.charAt(i+1);  
            sum =sum+ Math.abs(ascii1 - ascii2);
        }
        return sum;
    }
}